projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/defense/protection/magic.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:magic_protection",
+  "text": [
+    {
+      "text": "你不算是个好巫师，哈利，如果你老是用这个的话。多用用偏转咒语。"
```

```suggestion
      "text": "哈利，如果你老是得用这个的话，那你作为魔法师的水平还有待提高。多用用铁甲咒。"
```
语序啊语序，以及偏转咒语应该指的是铁甲咒（Protego），可以生成一道魔法屏障把攻击弹开，使其发生偏转。